static/css/perdiem.css

```diff
@@ -393,47 +362,195 @@ li.home-logo {
     padding: 0px;
 }
 
-.album-bio {    
-    margin: 0 auto;
-    padding-top: 10px;
-    padding-left: 10px;
-    padding-right: 10px;
-    max-width: 600px;
+.break-small {
+    padding-right: 30%;
+    padding-left: 30%;
 }
 
-.album-bio > ol {
+.track-listing {
+    margin: 0 auto;
+    padding: 10px;
     text-align: center;
-    list-style-position: inside;
 }
 
-.marketplace-container {
+.ol-center {
+    list-style-position:inside;
     text-align: center;
+}
+
+.bar-chart {
+    max-width: 600px;
+    margin: 0 auto;
     padding: 20px;
+    text-align: center;
 }
 
-.marketplace-icons {
-    width: 40px;
+#percentage {
+    color: #4cbd7e;
 }
 
-.break-small {
-    padding-left: 30%;
-    padding-right: 30%;
+#percentage1 {
+    color: #4cbd7e;
+    font-size: 18px;
 }
 
-.track-listing {
+.roi {
+    text-transform: uppercase;
+    color: #4d96ad;
+    font-family: Dosis;
+    font-size: 20px;
+    font-weight: 400;
+}
+
+.tabs-title > a {
+    position: vertical-align;
+    height: 100%;
+    padding: 5px;
+ }
+
+.tab-icon {
+    padding: 5px;
+    color: #4d96ad;
+ }
+
+.menu > li > a {
+    padding-top: 7px;
+    padding-right: 15px;
+    padding-bottom: 7px;
+    padding-left: 15px;
+ }
+
+a.nav-icon {
+    padding: 0px;
+ }
+
+li.home-logo {
+    padding: 2px;
+ }
+
+.name {
+    text-transform: uppercase;
+    color: #4d96ad;
+    font-family: Dosis;
+    font-size: 20px;
+}
+
+#myChart {
+    font-family: Dosis;
+}
+
+h2.nav-name {
+    margin: 0px;
+    padding: 5px;
     text-align: center;
-    margin: 0 auto;
+    font-size: 18px;
+}
+
+.login-item {
     padding: 10px;
 }
 
-.ol-center { 
-    text-align: center; 
-    list-style-position:inside;
+.left-align {
+    text-align: left
 }
 
-.bar-chart {
+.signup-manual {
+    padding-top: 15px;
+    padding-right: 12px;
+    padding-bottom: 15px;
+    padding-left: 12px;
+    color: #fff;
+    border-radius: 5px;
+    background-color: #85929E;
+    font-weight: 400;
+}
+
+#register-page {
+    position: relative;
+    height: 80vh;
+    background-color: black;
+    background-repeat: no-repeat;
+    background-size: cover;
+    }
+
+.signup-form {
+    margin-top: 10%;
+}
+
+.register-copy {
+    padding: 20px;
+    color: #285e6f;
+}
+
+.extra-padding-wide {
+    max-width: 800px;
     margin: 0 auto;
+    padding: 40px;
+}
+
+.contact-padding {
+    max-width: 700px;
+    margin: 0 auto;
+    padding: 40px;
+}
+
+.top-bar-left {
+    width: auto !important;
```

Comment:
I made a note on the first pass, I'm not sure if you saw it or not. The foundation.css is overriding the perdiem.css. There is no way to fix this unless using important or changing the foundation css. 

---

> Okay did a pretty good amount of updates to the CSS. Still couldn't figure out how to override a couple that are bing pulled from foundation, but I'll go back in a bit and take another shot. Let me know if there is anything that you see which is obvious that I'm missing.